packages/iTwinUI-react/src/core/Typography/Small/Small.tsx

```diff
@@ -28,7 +28,7 @@ export const Small = React.forwardRef<HTMLParagraphElement, SmallProps>(
     useTheme();
 
     return (
-      <p
+      <small
```

Comment:
Why images have changed?

Reply:
i think it's because small is an inline element, p is a block element